src/react-native/react-native-wizard.ts

```diff
@@ -0,0 +1,256 @@
+
+  clack.note(`To make sure everything is set up correctly, put the following code snippet into your application.
+The snippet will create a button that, when tapped, sends a test event to Sentry.
+
+After that check your project issues:
+
+${chalk.cyan(projectsIssuesUrl)}
+
+<Button title='Try!' onPress={ () => { Sentry.captureException(new Error('First error')) }}/>`);
+
+  const firstErrorConfirmed = clack.confirm({
+    message: `Have you successfully sent a test event?`,
+  });
+
+  return firstErrorConfirmed;
+}
+
+async function patchXcodeFiles({ authToken }: { authToken: string }) {
+  await addSentryCliConfig(authToken, {
+    ...propertiesCliSetupConfig,
+    name: 'source maps and iOS debug files',
+    filename: 'ios/sentry.properties',
+  });
+
+  if (platform() === 'darwin') {
+    await podInstall('ios');
+  }
+
+  const xcodeProjectPath = getFirstMatchedPath(XCODE_PROJECT);
+  if (!xcodeProjectPath) {
+    clack.log.warn(
+      `Could not find Xcode project file using ${chalk.bold(XCODE_PROJECT)}.`,
+    );
+    return;
+  }
+
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+  const xcodeProject = xcode.project(xcodeProjectPath);
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+  xcodeProject.parseSync();
+  const buildPhasesMap = getValidExistingBuildPhases(xcodeProject);
+
+  const bundlePhase = findBundlePhase(buildPhasesMap);
+  patchBundlePhase(bundlePhase);
+
+  const debugFilesUploadPhaseExists =
+    !!findDebugFilesUploadPhase(buildPhasesMap);
+  addDebugFilesUploadPhase(xcodeProject, { debugFilesUploadPhaseExists });
+
+  writeXcodeProject(xcodeProjectPath, xcodeProject);
+}
+
+async function patchAndroidFiles({ authToken }: { authToken: string }) {
+  await addSentryCliConfig(authToken, {
+    ...propertiesCliSetupConfig,
+    name: 'source maps and iOS debug files',
+    filename: 'android/sentry.properties',
+  });
+
+  const appBuildGradlePath = getFirstMatchedPath(APP_BUILD_GRADLE);
+  if (!appBuildGradlePath) {
+    clack.log.warn(
+      `Could not find Android ${chalk.bold(
+        'app/build.gradle',
+      )} file using ${chalk.bold(APP_BUILD_GRADLE)}.`,
+    );
+    return;
+  }
+
+  const appBuildGradle = fs.readFileSync(appBuildGradlePath, 'utf-8');
+  const includesSentry =
+    doesAppBuildGradleIncludeRNSentryGradlePlugin(appBuildGradle);
+  if (includesSentry) {
+    clack.log.warn(
+      `Android ${chalk.bold('app/build.gradle')} file already includes Sentry.`,
+    );
+    return;
+  }
+
+  const patchedAppBuildGradle = addRNSentryGradlePlugin(appBuildGradle);
+  if (doesAppBuildGradleIncludeRNSentryGradlePlugin(patchedAppBuildGradle)) {
+    clack.log.success(
+      chalk.greenBright(
+        `Added Sentry RN Gradle Plugin to ${chalk.bold('app/build.gradle')}.`,
```

Comment:
nit (sorry, feel free to ignore): Basically continuing my comment around colors, wdyt about removing `greenBright` and making the filename `cyan`? Our coloring guides aren't too specific around when to make a success message entirely green and when `clack.sucess` is enough. IMO it's okay to e.g. only make the final success message `greenBright` but feel free to go with whatever you prefer. (We're not super consistent here, yet, anyway.)

Reply:
No worries, the inconsistency is why I made all the messages green, as otherwise, only some were green which draws attention to them like they are more important but in this specific case it wasn't true.